testdata/picard/arrays/7991775143_R01C01_wo_optional_fields.vcf

```diff
@@ -179,6 +179,7 @@
 ##p95Green=4251
 ##p95Red=5479
 ##picardVersion=2.20.3-17-g8331bf3-SNAPSHOT
+##pipelineVersion=foo
```

Comment:
Since this is an optional parameter, we should have one test without pipelineVersion set in the VCF.  It should allow that case.